packages/router/src/lib/provide-file-router.ts

```diff
@@ -26,5 +28,10 @@ export function provideFileRouter(
       multi: true,
       useValue: () => updateMetaTagsOnRouteChange(),
     },
+    {
```

Comment:
I didn't know there was an internal token for adding root interceptor functions. I'd like to use this for the `requestContextInterceptor` also, but that's separate from this PR.